llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -4019,6 +4019,42 @@ class Function : public Constant {
 #endif
 };
 
+// Collector for SandboxIR related convenience functions that don't belong in
```

Comment:
Isn't it better to move the SandboxIRUtils into a sperate file perhaps in `SandboxIR/Utils.h` ?

Reply:
Done